src/ab-test-preview.js

```diff
@@ -12,16 +12,18 @@ document.querySelectorAll( '.ab-test-xb-preview' ).forEach( xb => {
 	const regex = new RegExp( `(utm_campaign|set_test)=test_xb_${ xbPostId }:(\\d+)`, 'i' );
 	const url_test = unescape( window.location.search ).match( regex );
 
-	for ( let i = 0; i < templates.length; i++ ) {
+	const total_templates = templates.length;
```

Comment:
Small note here - I know `url_test` came from some of my code but in JS we prefer to go for camelCase variable names these days just to match the way react likes to name things.

The main thing is always to try and maintain consistency with the existing code so use your judgment there.